### PR TITLE
Tag push support

### PR DESCRIPTION
**Pull request by Justinidlerz**

### CODE REVIEW

File: lib/webhooks.js [hunk 1]
```diff
@@ -35,18 +35,24 @@ function makeJob(project, config) {
...
+  if (payload.commits) {
+    const lastCommit = payload.commits[payload.commits.length - 1];
 
-  if(lastCommit.message.indexOf('[skip ci]') > -1) {
-    return {skipCi: true};
+    if(lastCommit.message.indexOf('[skip ci]') > -1) {
+      return {skipCi: true};
+    }
+  } else if (payload.after) {
+    if (/^0+$/.test(payload.after)) {
```
Comment (knownasilya) [inline lib/webhooks.js]: What is that regex testing? Can you leave a comment there for clarity?

File: lib/webhooks.js [hunk 2]
```diff
@@ -55,31 +61,53 @@ function startFromCommit(project, payload, send) {
...
+  let commit;
+  if (payload.commits) {
+    commit = payload.commits[payload.commits.length - 1];
+
+    if(!commit || !commit.author) {
+      debug('Unable to determine author from provided commit (may it\'s a merge commit?). Skipping');
+      debug(`Commit data extracted from payload:${util.inspect(commit)}`);
+      return null;
+    }
+  } else if (payload.user_name) {
```
Comment (knownasilya) [inline lib/webhooks.js]: Why does having a `user_name` signify that this is a tag? Please leave a comment explaining for future dev work. Thanks!

Comment (knownasilya) [reply lib/webhooks.js]: Especially since elsewhere you use `payload.ref.indexOf('refs/tags/')`

File: lib/webhooks.js (hunk 1 above)
Comment (Justinidlerz) [reply lib/webhooks.js]: okay

**Comment by knownasilya** [comment]
Thanks @Justinidlerz

**Pull request merged**